components/channel_header_dropdown/mobile_channel_header_dropdown.jsx

```diff
@@ -0,0 +1,143 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import {FormattedMessage} from 'react-intl';
+
+import StatusIcon from 'components/status_icon';
+import MobileChannelHeaderPlug from 'plugins/mobile_channel_header_plug';
+import {Constants} from 'utils/constants';
+import * as Utils from 'utils/utils';
+
+import * as MenuItem from './menu_items';
+
+export default class MobileChannelHeaderDropdown extends React.PureComponent {
+    static propTypes = {
+        user: PropTypes.object.isRequired,
+        channel: PropTypes.object.isRequired,
+        isDefault: PropTypes.bool.isRequired,
+        isFavorite: PropTypes.bool.isRequired,
+        isReadonly: PropTypes.bool.isRequired,
+        isArchived: PropTypes.bool.isRequired,
+        teammateId: PropTypes.string,
+        teammateStatus: PropTypes.string,
+    }
+
+    getChannelTitle = () => {
+        const {user, channel, teammateId} = this.props;
+
+        if (channel.type === Constants.DM_CHANNEL) {
+            const displayname = Utils.getDisplayNameByUserId(teammateId);
+            if (user.id === teammateId) {
+                return (
+                    <FormattedMessage
+                        id='channel_header.directchannel.you'
+                        defaultMessage='{displayname} (you)'
+                        values={{displayname}}
+                    />
+                );
+            }
+            return displayname;
+        }
+        return channel.display_name;
+    }
+
+    render() {
+        const {
+            user,
+            channel,
+            teammateStatus,
+            isDefault,
+            isFavorite,
+            isReadonly,
+            isArchived,
+        } = this.props;
+
+        return (
+            <div className='dropdown'>
+                {/* this.generateWebrtcIcon() */}
```

Comment:
Sure.

I haven't been able to find out how this icon is actually displayed, and I thought it duplicated with the Webrtc item in the drop-down menu, so didn't migrate that code.

I will remove that code if it would not be a problem.